src/lib/Behat/Page/UserSettingsPage.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace Ibexa\AdminUi\Behat\Page;
+
+use Behat\Mink\Session;
+use Ibexa\AdminUi\Behat\Component\ContentActionsMenu;
+use Ibexa\Behat\Browser\Element\Criterion\ElementTextCriterion;
+use Ibexa\Behat\Browser\Locator\VisibleCSSLocator;
+use Ibexa\Behat\Browser\Page\Page;
+use Ibexa\Behat\Browser\Routing\Router;
+
+class UserSettingsPage extends Page
+{
+    private ContentActionsMenu $contentActionsMenu;
+
+    public function __construct(Session $session, Router $router, ContentActionsMenu $contentActionsMenu)
+    {
+        parent::__construct($session, $router);
+        $this->contentActionsMenu = $contentActionsMenu;
+    }
+
+    public function verifyIsLoaded(): void
+    {
+        $this->contentActionsMenu->verifyIsLoaded();
+        $this->getHTMLPage()->find($this->getLocator('title'))->assert()->textEquals('User Settings');
+    }
+
+    public function switchTab(string $tabName): void
```

Comment:
As in UserPreferencesContext, maybe function here can be more precise.

Reply:
IMHO it's fine as it is currently: the usage is `$this->userSettingsPage->switchTab(...)`, which provides enough context about the action that's happening